godot-core/src/lib.rs

```diff
@@ -4,6 +4,63 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+//! Rust bindings for GDExtension, the extension API of [Godot](https://godotengine.org/) 4.
+//!
+//! TODO More documentation here...
```

Comment:
Let's remove this TODO, or at least move it upward to a normal (non-doc) comment.